src/utils/collection.ts

```diff
@@ -87,7 +76,7 @@ export async function makeCollectionFromFileList(fileList: FileList | File[]): P
   const collection: Collection<Uint8Array> = []
 
   for (let i = 0; i < fileList.length; i++) {
-    const file = fileList[i] as WebkitFile
+    const file = fileList[i] as File
```

Comment:
```suggestion
    const file = fileList[i]
```
The type cast is no longer needed in 4.5.4